manifests/c/codersnotes/VerySleepy/0.91.0.0/codersnotes.VerySleepy.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: codersnotes.VerySleepy
+PackageVersion: 0.91.0.0
+PackageLocale: en-US
+Publisher: codersnotes

```

```suggestion
Publisher: codersnotes.com
```
This must match Control Panel.